examples/spacing.rs

```diff
@@ -0,0 +1,80 @@
+use bevy::{prelude::*, render::texture::ImageSettings};
```

Comment:
Simple example based off of the layer example(just to show multiple tiles). The spacing is 8 pixels between each tile on both the X and Y axes.